bugbug/test_scheduling_features.py

```diff
@@ -108,3 +108,18 @@ def __call__(self, test_job, **kwargs):
                 "failures_past_56_pushes_in_components"
             ],
         }
+
+class arch(object):
```

Comment:
The implementation seems a bit too complex, e.g. we should be able to get the archs without using `linux`, `win`, `windows`, etc.